aidial_adapter_openai/utils/image_data_url.py

```diff
@@ -0,0 +1,39 @@
+import base64
+import re
+from typing import Optional
+
+from pydantic import BaseModel
+
+
+class ImageDataURL(BaseModel):
```

Comment:
But it represents image encoded as [data URL](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/Data_URLs).

URL part is quite important here. Otherwise, it's just a generic data.